R/tags.R

```diff
@@ -843,11 +843,15 @@ names(known_tags) <- known_tags
 #' @keywords NULL
 #' @import rlang
 tags <- lapply(known_tags, function(tagname) {
-  function(..., .noWS=NULL) {
+  # Make the function signature
+  tagFunc <- function(..., .noWS = NULL, .render = NULL) {}
+  # Overwrite the body with the `tagname` value injected into the body
+  rlang::fn_body(tagFunc) <- rlang::inject({rlang::expr({
     validateNoWS(.noWS)
     contents <- dots_list(...)
-    tag(tagname, contents, .noWS=.noWS)
-  }
+    tag(!!tagname, contents, .noWS=.noWS, .render = .render)
+  })})
+  tagFunc
```

Comment:
Would it make sense to use `rlang::new_function`?

Reply:
I could not get the arguments to include `...`. (Doesn't mean it can't be done. Just didn't figure it out.)

The body argument needs to be the result of the `inject` call.